<launch>
 <group ns="global_map">
  <node name="map_server" pkg="map_server" type="map_server" args="$(find global_path_planner)/map_data/global_map.yaml" >
  </node>
 </group>
 <group ns="plain_map">
  <node name="map_server" pkg="map_server" type="map_server" args="$(find global_path_planner)/map_data/plain_map.yaml" >
  </node>
 </group>
 <group ns="region_map">
  <node name="map_server" pkg="map_server" type="map_server" args="$(find global_path_planner)/map_data/region_map.yaml" >
  </node>
 </group>
  <node name="local_costmap" pkg="global_path_planner"  type="local_costmap" respawn="true"/>
  <node name="globa_path_publisher" pkg="global_path_planner" type="globa_path_publisher.py" args="final_path" respawn="true" />
  <node pkg="tf" type="static_transform_publisher" name="tf_static_pub" args="1.15 0 0 0 0 0 1 novatel velodyne 100" />
</launch>
